root_orchestrator/resource-abstractor/start-up.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# docker run -p 6379:6379 -d redis
+
+# create virtualenv
+virtualenv --clear -p python3.8 .venv
+source .venv/bin/activate
+
+.venv/bin/pip install -r requirements.txt
+
+export FLASK_ENV=development
+export FLASK_DEBUG=False # TRUE for logging
```

Comment:
Nitpick: Is this comment necessary? 😅 _(I get hammered at work regularly for my comments that are rather self-evident, so I start seeing this myself)_

Reply:
Totally agree with you there. I think there are a lot of repetitive files, ie start-up.sh & Dockerfile, and this is just a case of copy & paste.